src/people.rs

```diff
@@ -213,6 +224,22 @@ impl ContextPeopleExt for Context {
         initialized_value
     }
 
+    fn initialize_person_property<T: PersonProperty + 'static>(
+        &mut self,
```

Comment:
Is there a reason this needs to be a mutable reference?

Reply:
Hmm.... I was thinking the same reason set_person_property was.

Reply:
OK, so you don't need it to be `mut` but i think we should have it that way anyway, because it signals to callers that it actually changes things by intention

Reply:
I think it would be better for it not to be, since it would prevent you from calling it where you only have an immutable reference to context (maybe you want to do it lazily somewhere). The intention seems pretty clear to me from the name of the method

Reply:
You shouldn't be doing that. If you want to lazily initialize you should use an initializer.

The impact of `RefCell` is to hide the true semantics. That's OK in the lazy initialization because it has "as if" semantics but this doesnt'.